core/ppu_bg.go

```diff
@@ -94,64 +96,71 @@ func (ppu *PPU) bg34nba(data uint8) {
 	ppu.backgroundData.bg[3].tileSetBaseAddr = data >> 4
 }
 
+// 210Dh - 2114h horizontal and vertical background offset: https://forums.nesdev.com/viewtopic.php?t=15228 for the formula
+func (ppu *PPU) bgnhofs(bg uint8, data uint8) {
+	ppu.backgroundData.bg[bg].horizontalScroll = uint16(data&3)<<8 | uint16((ppu.backgroundData.PPU1ScrollLatch &^ 7)) | uint16(ppu.backgroundData.PPU2ScrollLatch&7)
+	ppu.backgroundData.PPU1ScrollLatch = data
+	ppu.backgroundData.PPU2ScrollLatch = data
+}
+
+func (ppu *PPU) bgnvofs(bg uint8, data uint8) {
+	ppu.backgroundData.bg[0].verticalScroll = uint16(data&3)<<8 | uint16(ppu.backgroundData.PPU1ScrollLatch)
```

Comment:
should this be : 
```suggestion
	ppu.backgroundData.bg[bg].verticalScroll = uint16(data&3)<<8 | uint16(ppu.backgroundData.PPU1ScrollLatch)
```

?